executor/write_test.go

```diff
@@ -1415,58 +1416,6 @@ func (s *testSuite) TestNullDefault(c *C) {
 	tk.MustQuery("select * from test_null_default").Check(testkit.Rows("<nil>", "1970-01-01 08:20:34"))
 }
 
-func (s *testSuite) TestGetFieldsFromLine(c *C) {
```

Comment:
Why move this test to other places?

Reply:
We do not need to expose GetFieldsFromLine only for test.